README.md

```diff
@@ -20,8 +23,16 @@ Another use case for writing plugins is importing data from sources other than f
 ### How to write an import plugin?
 Getting started is as easy as creating a new C# project in your favorite IDE, adding a plugin manifest and linking the [Zeiss.PiWeb.Sdk.Import](https://www.nuget.org/packages/Zeiss.PiWeb.Sdk.Import/) NuGet. To make this even easier, we provide a project template you can use as a starting point. More detailed information about this can be found in our [import plugin documentation](https://zeiss-piweb.github.io/PiWeb-Import-Sdk/).
 
+### Version compatibilities
+The following PiWeb versions are compatible with the below import SDK versions:
+
+| PiWeb version | Import SDK version |
+| - | - |
+| &ge; 2025.R1.1 | 1.0 |
+| &le; 8.6 | not supported |

```

Comment:
Mhh good question. I like the wildcard idea at least for the revision of the piweb version because this would express that the revision should and likely never will matter for the supported sdk version. So like this:

| PiWeb version | Import SDK version |
| - | - |
| &ge; 2025.R1.* | 1.0 |
| &le; 8.6.* | no plugin support |